src/network/modules/tokens/types.ts

```diff
@@ -61,6 +61,15 @@ export interface TokensRemoveExtendedParam {
   indices: AttrIndex[]
 }
 
+export interface TokensAddressMap {
+  [address: string]: bigint
+}
+
+export interface TokensMintBurnParam {
+  symbol: string
+  addresses: TokensAddressMap
```

Comment:
ManyJS will let you send multiple addresses and amounts even if Gwen doesn't yet.